ui/src/containers/VolumePage.js

```diff
@@ -31,27 +26,16 @@ const VolumePage = (props) => {
   const match = useRouteMatch();
   const currentVolumeName = match.params.name;
 
-  useEffect(() => {
-    if (currentVolumeName)
-      dispatch(fetchCurrentVolumeObjectAction(currentVolumeName));
-  }, [dispatch, currentVolumeName]);
-
-  useRefreshEffect(refreshNodesAction, stopRefreshNodesAction);
-  useRefreshEffect(refreshVolumesAction, stopRefreshVolumesAction);
-  useRefreshEffect(
-    refreshPersistentVolumesAction,
-    stopRefreshPersistentVolumesAction,
-  );
-  useRefreshEffect(
-    refreshCurrentVolumeStatsAction,
-    stopRefreshCurrentVolumeStatsAction,
-  );
+  useFetchCurrentVolumeObject(currentVolumeName);
+  useRefreshNodes();
```

Comment:
Can we drop `dispatch(fetchNodesAction());` below ?
Can we `const nodes = useRefreshNodes();` and drop `const nodes = useTypedSelector((state) => state.app.nodes.list);` below ?
I wonder the same for the other resources.

Reply:
we have actually an other tickets to handle the removal of all redux function